packages/edit-site/src/utils/constants.js

```diff
@@ -1,5 +1,42 @@
+/**
```

Comment:
Moving all this work over to:

- https://github.com/WordPress/gutenberg/pull/54484